src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -269,8 +277,14 @@ public void testExistingTargetFileXmlOutput() throws Exception {
     public void testExistingTargetFilePlainOutput() throws Exception {
 
         exit.checkAssertionAfterwards(() -> {
-            assertEquals(String.format(Locale.ROOT, "Starting audit...%n"
-                    + "Audit done.%n"), systemOut.getLog());
+            final LocalizedMessage auditStartMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, AUDIT_STARTED_MESSAGE,
+                    null, null, getClass(), null);
+            final LocalizedMessage auditFinishMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, AUDIT_FINISHED_MESSAGE,
+                    null, null, getClass(), null);
```

Comment:
@romani I advise against assigning these to a class field. These classes are based on static variable `locale` at https://github.com/checkstyle/checkstyle/blob/4a087c020c2b97130e94f9292b94cb4bc1f6ed93/src/main/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessage.java#L64 .
If we are changing it's values in a test's lifecycle, having it as a field will make it's values based on the time the class is instantiated and not when the test is run. If another test is manipulating any values, we could be stuck with improper values when the class is instantiated.